main.cc

```diff
@@ -1943,15 +1952,6 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
              */
             db.local().enable_autocompaction_toggle();
 
-            group0_service.start().get();
-            auto stop_group0_service = defer_verbose_shutdown("group 0 service", [&group0_service] {
-                sl_controller.local().abort_group0_operations();
-                group0_service.abort().get();
-            });
-
-            utils::get_local_injector().inject("stop_after_starting_group0_service",
-                [] { std::raise(SIGSTOP); });
-
             // Set up group0 service earlier since it is needed by group0 setup just below
             ss.local().set_group0(group0_service);
```

Comment:
Now this guy lives alone here without his friend `group0_service.start()`. Why not move him together with his friend?

Reply:
They are now together again <3